v2/src/core/components/TheAppFooter.vue

```diff
@@ -159,6 +171,20 @@ const socialIcons = [
         altText: 'Telegram icon'
     }
 ]
+
+const theme = useTheme()
+const isDarkMode = ref(false)
+
+const store = useStore()
+
+const toggleTheme = () => {
+    theme.global.name.value = theme.global.current.value.dark ? 'mainnetLightTheme' : 'mainnetDarkTheme'
+    store.setDarkMode(theme.global.name.value)
+}
+
+onMounted(() => {
+    isDarkMode.value = theme.global.name.value === 'mainnetDarkTheme'
+})
```

Comment:
you need to add a watch here somewhere. so that if value is changed on another tab, the theme used is consistent on the other ones.